src/components/Tabs/__tests__/tabs.test.tsx

```diff
@@ -0,0 +1,11 @@
+import {render, screen} from '@testing-library/react-native';
+import {Tabs} from '../index';
+
+test('Tabs component', async () => {
```

Comment:
the same, describe more concrete what are you expecting from this test. Start from "Should...."